### ISSUE TITLE
Accessibility standards

### ISSUE THREAD

**Issue by travisLilleberg** [projectblacklight/spotlight#2106]
The librarians and archivists I work with are wondering, is Spotlight section 508 compliant? What about WCAG 2 or 2.1?

Apologies if this is stated somewhere, but I couldn't find it.

**Comment by ggeisler** [comment]
@travisLilleberg I don't think we could say with confidence that Spotlight completely meets any of the accessibility standards you mention. I believe there was a pretty good review of Blacklight accessibility issues a few years ago (though I personally don't know whether all concerns were fixed or not), but I don't think anyone has thoroughly assessed the accessibility of all additional features Spotlight adds to Blacklight. I suspect there are areas where Spotlight doesn't completely meet the standards.
